### DISCUSSION TITLE
[@types/prompt] Missing `hidden` attribute in `RevalidatorSchema`

### DISCUSSION THREAD

**Discussion by longnguyen2004** [DefinitelyTyped/DefinitelyTyped#60754] [Issues with a @types package]
`prompt` schema accepts a `hidden` attribute to hide user inputs, but this is not in the `RevalidatorSchema` type

[a comment by an automated account is not shown]

**Reply by ffflorian** [reply]
Hi @longnguyen2004, I'd be happy if you could create a PR with your mentioned changes :)

**Comment by Semigradsky** [comment]
I have created PR: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/62494

**Reply by matthewberryman** [reply]
Thanks @Semigradsky for flagging this and submitting a PR, I am just taking a look now.
